src/main/java/com/cinema/servlets/MainServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.cinema.servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.Instant;
+
+
+public class MainServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        // if you remove line with date, you change nothing.
+        response.addDateHeader("Date", Instant.now().toEpochMilli());
+
+        response.setContentType("text/html");
```

Comment:
why do we need to set it manually?

Reply:
I don't know. I can delete this line, but the date will remain there.

Reply:
i do not get your answer. You have written the code and saying "i do not know what it is for" ?
Why did you set manually content type?

Reply:
Okay, there's no point in installing content type manually. It is installed automatically. So I'll just delete this line.